docs/guides/09-add-interactivity-and-events.md

```diff
@@ -321,10 +319,6 @@ Here it is the full example:
         };
         map.on('move', displayCenter);
 
-
-        //** CARTO VL functionality begins here **//
-
-
         // LAYER EVENTS & VARIABLES
         // Add layer as usual
```

Comment:
// LAYER EVENTS & VARIABLES	    
        // Add layer as usual	

seems like bad comments to me

Reply:
👍 removed!